### PR TITLE
added new app string

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
updated test for 429 dcc reissuance error

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13084

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -3381,6 +3381,8 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Reissuance_Error_Try_Again" = "Es ist ein Fehler bei der Aktualisierung Ihres Zertifikats aufgetreten. Bitte versuchen Sie es erneut. Wenn der Fehler weiterhin auftritt, kontaktieren Sie bitte die technische Hotline über App-Informationen -> Technische Hotline.";
 
+"HealthCertificate_Reissuance_Error_Rate_Limit" = "Das Zertifikat wurde bereits drei mal neu ausgestellt. Eine weitere Neuausstellung ist nicht möglich.<br />Wir empfehlen, Ihr bereits erneut ausgestelltes Zertifikat als PDF zu exportieren und auszudrucken. Damit können Sie es jederzeit erneut einscannen.";
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3384]: I don't think the `<br />` will be rendered as a line break, shouldn't we use a `\n` instead?

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/HTTPClient/Resources/CCLConfiguration/__tests__/Locator+CCLConfigurationTest.swift [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Locator_CCLConfigurationTest: XCTestCase {
 
 	func testGIVEN_Locator_WHEN_getUniqueIdentifier_THEN_IsSameAsKnowUniqueIdentifier() {
 		// GIVEN
-		let knownUniqueIdentifier = "b4f065feda7b87f0de440516d6edae6d1d3348d75eb8bba6c473d8ee6111fcfe"
+		let knownUniqueIdentifier = "0f24d96b87439f0b817b573407bc175f6a6c257991b02d87a3e1e53ef75ad11f"
```
[the remark this answers is not in the document]
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/HTTPClient/Resources/CCLConfiguration/__tests__/Locator+CCLConfigurationTest.swift:12]: my branch was startet from not yet merged work. The path of the resource got changed but the test was not updated. So I did to make thing build again.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3384]: Good point, I'll change it

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by nickguendling — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**